doc-output/configure-the-report-engine/assemblyreferences-element.md

```diff
@@ -49,11 +49,15 @@ The following sections describe attributes, child elements, and parent elements.
 | ------ | ------ |
 |Attributes|None|
 |Child Elements|<ul><li>__add__ - Optional element. Adds an assembly reference to the collection.</li><li>__clear__ - Optional element. Removes all references to inherited assembly names, allowing only the references that are added by the current add element.</li><li>__remove__ - Optional element. Removes a reference to an inherited assembly name from the collection.</li></ul>|
-|Parent Elements|<ul><li>__configuration__ - Specifies the root element in every configuration file that is used by the common language runtime and the.NET Framework applications.</li><li>__Telerik.Reporting__ - Configures all settings that Telerik Reporting Engine uses.</li></ul>|
+|Parent Elements|<ul><li>__configuration__ - Specifies the root element in every configuration file that is used by the common language runtime and the .NET Framework applications.</li><li>__Telerik.Reporting__ - Configures all settings that Telerik Reporting Engine uses.</li></ul>|
+
+When providing the Public Key Token, the Reporting Engine will try to match it with the Public Key Token of the found DLL. It will not load an assembly with the same name and a different Public Key Token and will throw an exception instead. If the Public Key Token is omitted in the configuration, the Reporting Engine will load the first DLL, which name matches the provided Assembly Name. Potentially, this may allow bad agents to replace trusted assemblies with malicious ones and constitutes a security vulnerability. The Public Key Token provides an extra security layer to your reports.
```

Comment:
I think there's an error in:
"the Reporting Engine will load the first DLL, **which** name matches the provided Assembly Name" 
I think it should be:
"the Reporting Engine will load the first DLL **whose** name matches the provided Assembly Name".
I'd suggest finish with something about "best practice", e.g. "Since PKT provides an extra security layer to your application (not reports!), we advise to always include it in the configuration file."